test/Microsoft.AspNetCore.Mvc.FunctionalTests/Infrastructure/MvcTestFixture.cs

```diff
@@ -22,14 +27,22 @@ protected MvcTestFixture(string solutionRelativePath)
         {
         }
 
-        protected override void ConfigureApplication(MvcWebApplicationBuilder<TStartup> builder)
+        protected override void ConfigureApplication(IWebHostBuilder builder)
```

Comment:
I think it's a little odd to have ConfigureApplication here but have it be the builder instead of the services or appbuilder. What if we mirrored how startup code looks?

/cc @davidfowl